stress/shell/src/main/java/alluxio/stress/cli/Benchmark.java

```diff
@@ -89,6 +91,7 @@ protected static void mainInternal(String[] args, Benchmark benchmark) {
     int exitCode = 0;
     try {
       String result = benchmark.run(args);
+      System.out.println(BENCHMARK_RESULT_TAG);
```

Comment:
Just catch one thing here, could you help me test whether the benchmark runs in cluster mode? This would probably break the task running in cluster mode since we are reading from the printout in cluster mode.

Reply:
@jja725 done.I had tested in `--cluster` mode.